src/main/primary_entry/script/page/map/RangeInput.js

```diff
@@ -94,7 +94,7 @@ function internalHandleSliderMoved() {
  * Update the range slider value;
  */
 function labelValueChanged() {
-    let newValue = this.label.val();
+    let newValue = parseInt(this.label.val());
```

Comment:
Silly Javascript thought (for example) 75 > 350 because it was doing a string compare below. That's why most 1-2 digit range input was automatically being reset to the max.

Reply:
Hey at least it wasn't using octal!